data/secscan_model/secscan_v4_model.py

```diff
@@ -307,6 +314,21 @@ def mark_manifest_unsupported(manifest):
                     metadata_json={},
                 )
 
+        def mark_manifest_layer_too_large(manifest):
+            with db_transaction():
+                ManifestSecurityStatus.delete().where(
+                    ManifestSecurityStatus.manifest == manifest._db_id,
```

Comment:
isin't `manifest_id` unique across? Do we need the extra `AND` with the `repository_id`?

Reply:
It _should_ be. I didn't change the existing query we had for unsupported manifests, so I think it should be fine. I don't think the extra AND clause should have a performance impact on the overall query.